### PR TITLE
[incubator/schema-registry] Updating GroupID, and moving to using kafka coordinator master election

### PR DESCRIPTION
**Pull request by bradenwright**
**What this PR does / why we need it**:
Updates GroupID to default to release name but can be overridden.  This allows each Release to be its own group of schema registry workers by default.  Also using Kafka Coordinator Master Election instead of ZooKeeper.

Details: https://docs.confluent.io/current/schema-registry/docs/design.html#kafka-coordinator-master-election

@benjigoldberg

### CODE REVIEW

File: incubator/schema-registry/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 name: schema-registry
 home: https://docs.confluent.io/current/schema-registry/docs/index.html
-version: 0.1.1
+version: 0.1.2
```
Comment (timstoop) [inline incubator/schema-registry/Chart.yaml]: Please update the minor, as you're adding new features.

**Review by timstoop — CHANGES_REQUESTED** [review]

**Comment by benjigoldberg** [comment]
@bradenwright can you bump the version? We're good to go once you've done that

**Comment by alexander-alvarez** [comment]
Since this change, https://github.com/kubernetes/charts/tree/master/incubator/schema-registry#parameters (`zookeeperUrl`, and `zookeeperPort`) have been out of sync.

Similarly https://github.com/kubernetes/charts/blob/master/incubator/schema-registry/templates/_helpers.tpl#L18..L29 is not longer used, so should be removed

**Comment by benjigoldberg** [comment]
@bradenwright can you have a look at this ^^

**Comment by bradenwright** [comment]
@alexander-alvarez what do you mean by out of sync.  I believe the parameters you are referring to have been removed.  If you feel that both methods of electing a master should be supported, to prevent errors in upgrading or something of that nature, I'm open.  If you can provide a little more detail we can figure something out.

Nice catch on unused code, I'll address that when we resolve other issue you are having.

**Comment by alexander-alvarez** [comment]
@bradenwright The parameters have been removed, but they're still in the documentation. So the code is out of sync with the documentation. Nothing sophisticated and challenging :smiley:

**Comment by bradenwright** [comment]
@alexander-alvarez fyi, https://github.com/kubernetes/charts/pull/5352 nice catch, 💯don't want to leave unused code and documentation around.

**Pull request merged**